fdbclient/TenantManagement.actor.cpp

```diff
@@ -66,4 +67,30 @@ int64_t extractTenantIdFromKeyRef(StringRef s) {
 	return TenantAPI::prefixToId(prefix, EnforceValidTenantId::False);
 }
 
+// validates whether the lastTenantId and the nextTenantId share the same 2 byte prefix
+bool nextTenantIdPrefixMatches(int64_t lastTenantId, int64_t nextTenantId) {
+	if (getTenantIdPrefix(nextTenantId) != getTenantIdPrefix(lastTenantId)) {
+		TraceEvent(SevWarnAlways, "TenantIdPrefixMismatch")
```

Comment:
Since this is going to leave the cluster unable to add new tenants, I'd say let's upgrade this to a `SevError`. This is mostly just meant to be a slightly nicer form of ASSERT.

Reply:
I guess we could make the severity different in simulation and the real world, which is probably good enough for this

Reply:
Ah i completely forgot about that thanks